go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	golang.org/x/oauth2 v0.4.0
 	golang.org/x/sync v0.1.0
 	google.golang.org/api v0.103.0
+	google.golang.org/grpc v1.53.0
```

Comment:
Unsure if this was forgotten or specific to my setup - happy to remove it if undesired.

Reply:
Looks like you are correct. This might be a miss on previous prs